rules/private/phases/phase_test_launcher.bzl

```diff
@@ -19,19 +19,19 @@ def phase_test_launcher(ctx, g):
     files = ctx.attr._jdk[java_common.JavaRuntimeInfo].files.to_list() + [g.compile.zinc_info.apis]
 
     coverage_replacements = {}
-    coverage_runner_jars = []
+    coverage_runner_jars = depset(direct = [])
```

Comment:
Why do we need to use `depset` in this file in general?

Reply:
These were ending up in a depset, anyway, so I made them depsets. 

`transitive_runtime_deps` is a depset, so that I either need to `to_list()` or keep as a depset. I don't have strong opinions either way.

Reply:
I do: https://docs.bazel.build/versions/master/skylark/performance.html#avoid-calling-depsetto_list